mne/io/bti/bti.py

```diff
@@ -953,7 +953,7 @@ def _read_bti_header(pdf_fname, config_fname, sort_by_ch_name=True):
         info['order'] = sort_by_name_idx
     else:
         info['chs'] = chans
-        info['order'] = Ellipsis
+        info['order'] = None
```

Comment:
if you make this np.arange(len(chans))` then you don't need the conditional later, right?